ui/src/app/components/webcomponents/select-test-lab.component.ts

```diff
@@ -125,7 +127,10 @@ export class SelectTestLabComponent implements OnInit {
     this.integrationsService.findAll().subscribe(res => {
       this.applications = res;
       if(this.isNewTestPlan) {
-        if (!this.isTestsigmaLabInstalled) {
+        if(this.executionEnvironment){
```

Comment:
this code block can be simplified like 
set default value for a var and overwrite it with intended condition.
setValue at the end with updated value
@manikanta-reddy-t

Reply:
can we improve this if, else block @manikanta-reddy-t ?